Makefile

```diff
@@ -6,7 +6,7 @@ PYTEST_CI_ARGS := --cov=. --cov-report=term-missing
 
 NPROCS = `nproc`
 
-CUDA_ARCH := "sm_70"
+CUDA_ARCH := "70"
```

Comment:
Would be good to support the sm_ prefix and strip it off it is provided. Don't know that we will fix this everywhere immediately and its easy to solve with some sed
```
echo sm_75 | sed -e "s/^sm_//"
```

Reply:
I like it. Done in abe588d.